core/src/rust/tantivy_utils/src/query/bitset_weight.rs

```diff
@@ -0,0 +1,90 @@
+//! Weight adapter for a cached bitset
```

Comment:
I guess we can syncup offline for understand more about these tantivy internals.